src/interface/drawstatus.cpp

```diff
@@ -480,18 +497,33 @@ void drawStatus()
 		{
 			color = SDL_MapRGB(mainsurface->format, 128, 0, 0);
 		}
+
+		// Only draw the actual Health bar if the Player is alive
 		drawRect(&pos, color, 255);
 	}
+
+	// Print out the amount of HP the Player currently has
 	snprintf(tempstr, 4, "%d", stats[clientnum]->HP);
 	printTextFormatted(font12x12_bmp, 96 - strlen(tempstr) * 6, yres - 16 - 64 - 6, tempstr );
 
-	// magic
+	// PLAYER MAGIC BAR
+	// Display the Magic bar border
 	pos.x = 12;
 	pos.w = 39;
 	pos.h = 156;
 	pos.y = yres - 168;
 	drawTooltip(&pos);
+
+	// Display "MP" at the top of Magic bar
 	ttfPrintText(ttf12, pos.x + 8, pos.y + 6, language[307]);
+
+	// Display border between actual Magic bar and "MP"
+	pos.x = 12;
+	pos.w = 39;
+	pos.h = 1;
```

Comment:
Change height from 1 to 0 - Makes the border 2px high instead of 3px. Matches the 2px border for the rest of the bar.